### PR TITLE
Rename types

### PR DESCRIPTION
**Pull request by aiwilliams**

### CODE REVIEW

File: docs/jupiterone-io/index.md [hunk 1]
```diff
@@ -2,35 +2,36 @@
...
-| Account                 | `airwatch_account`    : `Account`           |
-| Admin                   | `airwatch_user`       : `Admin`             |
-| OrganizationGroup       | `airwatch_user_group` : `OrganizationGroup` |
-| Device                  | `airwatch_device`     : `Device`            |
-| User                    | `endpoint_user`       : `User`              |
+| Example Entity Resource | \_type : \_class of the Entity         |
+| ----------------------- | -------------------------------------- |
+| Account                 | `airwatch_account` : `Account`         |
+| Admin                   | `airwatch_user` : `User`               |
+| OrganizationGroup       | `airwatch_group` : `OrganizationGroup` |
```
Comment (aiwilliams) [inline docs/jupiterone-io/index.md]: These can contain admins, users, and devices, so let's not type them as user groups.

File: docs/jupiterone-io/index.md [hunk 2]
```diff
@@ -2,35 +2,36 @@
...
+| Device                  | `airwatch_device` : `Device`           |
+| User                    | `device_user` : `User`                 |
```
Comment (aiwilliams) [inline docs/jupiterone-io/index.md]: This matches other MDM integrations.

File: docs/jupiterone-io/index.md [hunk 3]
```diff
@@ -2,35 +2,36 @@
...
-| Admin                   | `airwatch_user`       : `Admin`             |
-| OrganizationGroup       | `airwatch_user_group` : `OrganizationGroup` |
-| Device                  | `airwatch_device`     : `Device`            |
-| User                    | `endpoint_user`       : `User`              |
+| Example Entity Resource | \_type : \_class of the Entity         |
+| ----------------------- | -------------------------------------- |
+| Account                 | `airwatch_account` : `Account`         |
+| Admin                   | `airwatch_user` : `User`               |
+| OrganizationGroup       | `airwatch_group` : `OrganizationGroup` |
+| Device                  | `airwatch_device` : `Device`           |
```
Comment (aiwilliams) [inline docs/jupiterone-io/index.md]: We need to look at having types specific to the endpoint device, such as `computer`, `tablet`, `smartphone`, `iphone`, `ipad` etc. We'd like to be as specific as possible.

**Comment by aiwilliams** [comment]
@eXtremeX I'm making some changes before pushing this to production, thought you'd be interested in the reasons for the specific changes.

File: docs/jupiterone-io/index.md (hunk 1 above)
Comment (erkangz) [reply docs/jupiterone-io/index.md]: the class should be `UserGroup`

Comment (erkangz) [reply docs/jupiterone-io/index.md]: i misread your comment.  it that case, it should probably be `['Group', 'UserGroup']`?

Comment (aiwilliams) [reply docs/jupiterone-io/index.md]: Doh! 🤦‍♂

There can be more than just `User` entities related to the group, so perhaps I should do `["Group", "UserGroup"]` (see https://support.jupiterone.io/hc/en-us/articles/360022903573-Data-Model-Overview)?

File: docs/jupiterone-io/index.md (hunk 3 above)
Comment (erkangz) [reply docs/jupiterone-io/index.md]: for the time being, how about we use `user_endpoint` to be consistent with other integrations?

File: docs/jupiterone-io/index.md (hunk 1 above)
Comment (aiwilliams) [reply docs/jupiterone-io/index.md]: Ah, good, looks like our comments crossed on the InterWebs.

File: docs/jupiterone-io/index.md (hunk 3 above)
Comment (aiwilliams) [reply docs/jupiterone-io/index.md]: Sounds good, updated.

**Pull request merged**